doc/apimanager-reference.md

```diff
@@ -259,6 +264,8 @@ that should be set on it.
 | **Field** | **json/yaml field**| **Type** | **Required** | **Default value** | **Description** |
 | --- | --- | --- | --- | --- | --- |
 | StorageClassName | `storageClassName` | string | No | nil | The Storage Class to be used by the PVC |
+| Resources | `resources` | [PersistentVolumeClaimResourcesSpec](#PersistentVolumeClaimResourcesSpec) | No | nil | The minimum resources the volume should have |
+| VolumeName | `volumeName` | string | No | nil | The binding reference to the PersistentVolume backing this claim |
```

Comment:
Can we clarify that this is used to provide a pre-existing PV instead of letting the operator create its own one?